crates/tests-e2e/tests/e2e_close_position.rs

```diff
@@ -89,6 +90,34 @@ async fn can_open_close_open_close_position() {
         .unwrap();
     tracing::info!(%app_off_chain_balance, "Opened second position");
 
+    // rolling over before closing the second position reproduces https://github.com/get10101/10101/issues/2126
```

Comment:
:upside_down_face: Probably controversial, but could we add a dedicated test? I suppose the problem would happen with the first position too. It's a bit of an arbitrary spot to do this.

Reply:
Also, I think the test is no longer reproducing the issue because the issue is already fixed by now. Can you adapt the comment? I was a bit confused with the wording.